web/template/mlpa-documents.gohtml

```diff
@@ -0,0 +1,64 @@
+{{ template "page" . }}
+
+{{ define "title" }}Documents{{ end }}
+
+{{ define "main" }}
+    <div class="govuk-grid-row" id="documents">
+        <div class="govuk-grid-column-two-thirds">
+
+            {{ template "mlpa-header" (caseTabs .Lpa "documents") }}
+
+            <div class="govuk-!-display-block">
+                <h1 class="govuk-heading-l govuk-!-margin-bottom-5 govuk-!-display-inline govuk-!-padding-right-4">Documents</h1>
+                <button class="govuk-button govuk-button--secondary" data-module="govuk-button">Create a document</button>
+            </div>
+
+            <a class="govuk-link govuk-!-padding-right-4" href="#">
+                Download
+                <span class="govuk-visually-hidden">Download selected documents</span>
+            </a>
+            <a class="govuk-link app-!-colour-text-red" href="#">Delete</a>
```

Comment:
These should be wrapped inside a `govuk-body` div, otherwise their font size is unpredictable. Also the visually hidden text is read out with the context, so need to avoid it saying "Download" twice:
```suggestion
            <div class="govuk-body">
                <a class="govuk-link govuk-!-padding-right-4" href="#">
                    Download
                    <span class="govuk-visually-hidden">selected documents</span>
                </a>
                <a class="govuk-link app-!-colour-text-red" href="#">
                    Delete
                    <span class="govuk-visually-hidden">selected documents</span>
                </a>
            </div>
```